src/circuit/gadgets/circom/mod.rs

```diff
@@ -0,0 +1,26 @@
+//! # Usage of circom coprocessors.
```

Comment:
I would advise putting `#[deny(missing_docs)]` at the top of the module and fixing the errors.